components/navigation/MenuBlocks.js

```diff
@@ -10,9 +11,8 @@ export default function MenuBlocks ({
         items.map((item, index) => {
           const isExternalHref = item.href && item.href.startsWith('http');
           return (
+            <Link href={item.comingSoon ? '' : item.href} key={index}>
```

Comment:
This change has been made in order to improve the routing of the pages. The links whose href is a slug (or internal link pointing to a page in a website) doesn't loads the tab on clicking, instead renders the page quickly using `next/router`.

Reply:
I think that it's better to have only `a` tag for external ones. Please check first if `href` starts by `http(s)` and only if yes, apply the `Link` component. Don't base logic on `comingSoon`prop.